FRONTEND/visualElement.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { View, Image, ImageSourcePropType, StyleSheet } from 'react-native';
+import Animated from 'react-native-reanimated';
+import LottieView from 'lottie-react-native';
+
+interface VisualElementProps {
```

Comment:
1. use type instead of interface while mentioning Props
2. Document the type per parameter to have clarity which parameter is mapped to which quantity for the visual element